polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1470,10 +1503,19 @@ pub mod migrations {
 		type PalletName = TipsPalletName;
 	}
 
+	/// Upgrade Session keys to exclude `ImOnline` key.
+	/// When this is removed, should also remove `OldSessionKeys`.
+	pub struct UpgradeSessionKeys;
+	impl frame_support::traits::OnRuntimeUpgrade for UpgradeSessionKeys {
+		fn on_runtime_upgrade() -> Weight {
+			Session::upgrade_keys::<OldSessionKeys, _>(transform_session_keys);
+			Perbill::from_percent(50) * BlockWeights::get().max_block
```

Comment:
Does this value make sense? It's copy-pasted from the previous session keys migration, and I'm not sure how it should be calculated.

Reply:
Yeah fine.